lib/AbstractConverter.php

```diff
@@ -70,152 +73,260 @@ public function __construct(NamingStrategy $namingStrategy)
-        });
-        $this->addAliasMap("http://www.w3.org/2001/XMLSchema", "normalizedString", function (Type $type)
-        {
-            return "string";
-        });
-        $this->addAliasMap("http://www.w3.org/2001/XMLSchema", "integer", function (Type $type)
-        {
-            return "integer";
-        });
-        $this->addAliasMap("http://www.w3.org/2001/XMLSchema", "int", function (Type $type)
-        {
-            return "integer";
-        });
-        $this->addAliasMap("http://www.w3.org/2001/XMLSchema", "unsignedInt", function (Type $type)
-        {
-            return "integer";
-        });
-        $this->addAliasMap("http://www.w3.org/2001/XMLSchema", "negativeInteger", function (Type $type)
-        {
-            return "integer";
-        });
-        $this->addAliasMap("http://www.w3.org/2001/XMLSchema", "positiveInteger", function (Type $type)
-        {
-            return "integer";
-        });
-        $this->addAliasMap("http://www.w3.org/2001/XMLSchema", "nonNegativeInteger", function (Type $type)
-        {
-            return "integer";
-        });
-        $this->addAliasMap("http://www.w3.org/2001/XMLSchema", "nonPositiveInteger", function (Type $type)
-        {
-            return "integer";
-        });
-        $this->addAliasMap("http://www.w3.org/2001/XMLSchema", "long", function (Type $type)
-        {
-            return "integer";
-        });
-        $this->addAliasMap("http://www.w3.org/2001/XMLSchema", "unsignedLong", function (Type $type)
-        {
-            return "integer";
-        });
-        $this->addAliasMap("http://www.w3.org/2001/XMLSchema", "short", function (Type $type)
-        {
-            return "integer";
-        });
-        $this->addAliasMap("http://www.w3.org/2001/XMLSchema", "boolean", function (Type $type)
-        {
-            return "boolean";
-        });
-        $this->addAliasMap("http://www.w3.org/2001/XMLSchema", "nonNegativeInteger", function (Type $type)
-        {
-            return "integer";
-        });
-        $this->addAliasMap("http://www.w3.org/2001/XMLSchema", "positiveInteger", function (Type $type)
-        {
-            return "integer";
-        });
-        $this->addAliasMap("http://www.w3.org/2001/XMLSchema", "language", function (Type $type)
-        {
-            return "string";
-        });
-        $this->addAliasMap("http://www.w3.org/2001/XMLSchema", "token", function (Type $type)
-        {
-            return "string";
-        });
-        $this->addAliasMap("http://www.w3.org/2001/XMLSchema", "anyURI", function (Type $type)
-        {
-            return "string";
-        });
-        $this->addAliasMap("http://www.w3.org/2001/XMLSchema", "byte", function (Type $type)
-        {
-            return "string";
-        });
-        $this->addAliasMap("http://www.w3.org/2001/XMLSchema", "duration", function (Type $type)
-        {
-            return "DateInterval";
-        });
+        $this->addAliasMap(
```

Comment:
If the line is shorter than 120 chr, i prefer to have function calls on a single line.

multi-line call are really "noisy" to me...